PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/applicationinsights/ApplicationInsightsResourceRegistryEclipse.java

```diff
@@ -10,6 +10,7 @@
 import java.util.List;
 
 import com.microsoft.azure.toolkit.lib.common.model.Subscription;
+import com.microsoft.azuretools.authmanage.models.SubscriptionDetail;
```

Comment:
Why we introduce the old subscription model here?

Reply:
will remove it later